pages/common/gnmic.md

```diff
@@ -0,0 +1,25 @@
+# gnmic
+
+> A gNMI command-line client.
+> Manage gNMI network device configuration and view operational data.
+> More information: <https://gnmic.kmrd.dev>.
+
+- Request device capabilities:
+
+`gnmic --address {{ip:port}} --insecure capabilities`
```

Comment:
Question: we use `--insecure` as a flag for all of these examples but don't use them in other tldr pages. Is it worthwhile to have a single example explaining why this flag would be used, and then drop the flag from other examples?

Reply:
I removed the `--insecure` flag mentions.